src/main/resources/application.yml

```diff
@@ -2,6 +2,8 @@
 spring:
   profiles:
     active: @activatedSpringProfile@
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
can we include a comment on which bean we are overriding for the sake of this setting